src/scss/views/widgets/_pill.scss

```diff
@@ -0,0 +1,59 @@
+.pill {
+  display: flex;
+  align-items: center;
+  width: fit-content;
+  height: 32px;
+  border-radius: $border-radius-large;
+  font-size: 14px;
+
+  .pill-title {
+    font-weight: $font-weight-semibold;
+    color: $color-white;
+    margin: 0px 16px;
+  }
+
+  &.removeable {
+    position: relative;
+
+    &:hover {
+      cursor: pointer;
+      color: $color-white;
+      font-weight: $font-weight-semibold;
+
+      .pill-title {
+        margin-left: 8px;
+        margin-right: 24px;
+      }
+
+      &::after {
+        content: 'X';
```

Comment:
is this just til the icons are ready?

Reply:
Doesn't matter one way or the other, can be done either way. Not a priority.